Source/ACE/Entity/Landblock.cs

```diff
@@ -165,6 +167,37 @@ public void StartUseTime()
             new Thread(UseTime).Start();
         }
 
+        private void AddPlayerTracking(List<WorldObject> wolist, Player player)
+        {
+            Parallel.ForEach(wolist, (o) =>
+            {
+                if (o.Guid.IsCreature())

```

Comment:
could be simplified:
if (!o.Guid.IsCreature() || (o.Guid.IsCreature && (o as Creature).IsAlive)
    player.TrackObject(o);